Foundation/NSJSONSerialization.swift

```diff
@@ -23,6 +23,7 @@ extension JSONSerialization {
         public static let mutableContainers = ReadingOptions(rawValue: 1 << 0)
         public static let mutableLeaves = ReadingOptions(rawValue: 1 << 1)
         public static let allowFragments = ReadingOptions(rawValue: 1 << 2)
+        internal static let useReferenceNumericTypes = ReadingOptions(rawValue: 1 << 3)
```

Comment:
We're unlikely to need to add _that_ many options; in the future, we can modify this value if need be, since it's just internal here.